[new product]  SUSE Manager (#5088)
captn3m0/endoflife.date: products/suse-manager.md

---
title: SUSE Manager
category: server-app

> tags: os suse

Reviewer: Not an operating system as far as I can tell.
```suggestion
tags: suse
```

iconSlug: suse
permalink: /suse-manager
alternate_urls:

> -   /suma

Reviewer: seems more confusing to be worth it. https://github.com/suma-project/Suma for eg. Easier to avoid.
```suggestion
```

> versionCommand: cat /etc/uyuni-release

Reviewer: The uyuni project recommends `zypper info Uyuni-Server-release` in their new issue template, and I'm guessing there's a similar package name we can use for suse-manager instead?

Author: Good idea. I had a look at a SUMA 4.3 system. The corresponding command would be `zypper info SUSE-Manager-Server-release`. The current `cat` command would print the Uyuni release that the SUMA release is based on - so definitely not what we're looking for. Will this this - thanks for pointing out.

Author: Fixed in commit b43b21733529c8b300ef31d93c7fa855835de3c0 - what do you think?

releasePolicyLink: https://www.suse.com/lifecycle
changelogTemplate: "https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/{{'__RELEASE_CYCLE__'}}/"

> releaseColumn: false

Reviewer: Since we have `latest` values for all rows, this should be visible.

```suggestion
releaseColumn: true
```

releaseDateColumn: true
eolColumn: General Support

releases:
-   releaseCycle: "4.3"
    releaseDate: 2022-06-20

>     eol: false

Reviewer: As per https://www.suse.com/lifecycle#suse-manager-43-server

```suggestion
    eol: 2025-06-30
```

    latest: '4.3.11.1'
    latestReleaseDate: 2024-03-07

-   releaseCycle: "4.2"
    releaseDate: 2021-06-21
    eol: 2023-10-31
    latest: '4.2.14'
    latestReleaseDate: 2023-08-29

-   releaseCycle: "4.1"
    releaseDate: 2020-07-21
    eol: 2022-10-31
    latest: '4.1.15'
    latestReleaseDate: 2022-06-20

-   releaseCycle: "4.0"
    releaseDate: 2019-06-26
    eol: 2021-06-30
    latest: '4.0.14'
    latestReleaseDate: 2021-06-22

-   releaseCycle: "3.2"
    releaseDate: 2018-06-25
    eol: 2020-09-30
    latest: '3.2.15'
    latestReleaseDate: 2020-08-19

-   releaseCycle: "3.1"
    releaseDate: 2017-06-27
    eol: 2019-09-30
    latest: '3.1.11'
    latestReleaseDate: 2019-03-01

-   releaseCycle: "3.0"
    releaseDate: 2016-04-05
    eol: 2018-10-31
    latest: '3.0.12'
    latestReleaseDate: 2018-06-01

-   releaseCycle: "2.1"
    releaseDate: 2014-06-02
    eol: 2017-04-30
    latest: '2.1.19'
    latestReleaseDate: 2016-10-31

-   releaseCycle: "1.7"
    releaseDate: 2012-08-09
    eol: 2015-06-30
    latest: '1.7.6'
    latestReleaseDate: 2013-08-26

---

> > [SUSE Manager](https://www.suse.com/products/suse-manager/) is an open source infrastructure management solution designed to simplify and secure your entire mixed Linux environment - at the core, on the edge or in the cloud(s). Maintain infrastructure security & compliance at any scale - from 10 to 1 million clients - from a single console.

Reviewer: Some small suggestions:

1. Switch to a neutral third-person voice, as per our [Style Guide](https://github.com/endoflife-date/endoflife.date/wiki/Style-Guide#tone-and-text)
2. Mention Salt as the engine underneath it, which seems helpful. I come from the IaaS world, and it helps me contextualize what SUSE Manager does immediately.
3. Change the second sentence to be less promotional, and more informational. 

```suggestion
> [SUSE Manager](https://www.suse.com/products/suse-manager/) is an open source infrastructure management solution designed to simplify and secure a mixed Linux environment. It uses the Salt configuration management system to manage clients, deploy patches and packages, and report changes at scale.
```

> SUSE Manager is usually released every 12 months. The current version is usually supported for additional 12 months once a successor was published. The upstream open-source project is [Uyuni](https://uyuni-project.org) - it offers the same functionality, but comes without paid support.

Reviewer: Switch the OSS mention to a warning, similar to how we do it on https://endoflife.date/vmware-harbor-registry. Would have just linked to /uyuni, but since we don't track Uyuni yet - mentioning a one-liner for their release policy seems helpful.

Slightly reworded the policy summary to be easier to read.

```suggestion

{: .warning }
> This page tracks SUSE Manager, which is a SUSE product based on the [open-source Uyuni project](https://uyuni-project.org). Uyuni offers the same functionality, but follows a rolling release - bugfixes are only available on the latest version.

SUSE Manager is usually released every 12 months. The current release is usually supported for an additional 12 months once after the next release, resulting in a 24 month support cycle. SUSE Manager Proxy follows the same lifecycle as SUSE Manager Server.
```

Author: This is great! Thanks for pointing out.

---
title: SUSE Manager
category: server-app
tags: suse
iconSlug: suse
permalink: /suse-manager
versionCommand: zypper info SUSE-Manager-Server-release
releasePolicyLink: https://www.suse.com/lifecycle

> changelogTemplate: "https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/{{'__RELEASE_CYCLE__'}}/"

Reviewer 2: ```suggestion
changelogTemplate: https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/__RELEASE_CYCLE__/
```

Can be simplified.

releaseColumn: true
releaseDateColumn: true
eolColumn: General Support

> releases:

Reviewer 2: ```suggestion
releases:
-   releaseCycle: "5.0"
    releaseDate: 2024-07-12
    eol: 2026-06-30
    latest: '5.0.0'
    latestReleaseDate: 2024-07-12
```

Based on https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/5.0/index.html.

Author: Good one, thanks for taking are of it!

> releases:

Reviewer 2: Looked at https://github.com/endoflife-date/endoflife.date/pull/5088#pullrequestreview-2161539612 and I have a similar question : where can endoflife.date maintainers find the latest releases and their dates.

In each released notes we have the list of all versions, but only a single date at the document level. Is it what we should use for the latest release date ?

Author: Yes, exactly. I wasn't able to find a nicer location where to get these version information.

-   releaseCycle: "4.3"
    releaseDate: 2022-06-20
    eol: 2025-06-30

>     latest: '4.3.11.1'

Reviewer 2: Looks like some latest releases must be updated.

Author: Good one, 4.3.13 was missing - I fixed it with commit 818563d89e486b9ceb394c3a54c5893adae53e22. What do you think?

---
title: SUSE Manager
category: server-app
tags: suse
iconSlug: suse
permalink: /suse-manager
versionCommand: zypper info SUSE-Manager-Server-release
releasePolicyLink: https://www.suse.com/lifecycle
changelogTemplate: https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/__RELEASE_CYCLE__/
releaseColumn: true
releaseDateColumn: true
eolColumn: General Support

> releases:

Reviewer 2: ```suggestion
# Release dates are documented in each release notes in the "Version revision history" paragraph.
releases:
```

-   releaseCycle: "5.0"
    releaseDate: 2024-07-12
    eol: 2026-06-30
    latest: '5.0.0'
    latestReleaseDate: 2024-07-12

-   releaseCycle: "4.3"
    releaseDate: 2022-06-20
    eol: 2025-06-30
    latest: '4.3.13'

>     latestReleaseDate: 2024-06-01

Reviewer 2: ```suggestion
    latestReleaseDate: 2024-07-04
```

https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.3/index.html

Author: Ah, now I understand what you mean! 💡

I think the address on the top only refers to the last document render job. Sometimes they might also fix typos or append notes. I assume that the version history represents the appropriate release date, see here: https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.3/index.html#_version_revision_history

So, 2024-06-10 for the very latest 4.3.13. What do you think, does this sound plausible? 🙂

Reviewer 2: Oh, right, I missed it. Thanks for pointing this out. I will review the dates accordingly and document this for maintainers.

-   releaseCycle: "4.2"
    releaseDate: 2021-06-21
    eol: 2023-10-31
    latest: '4.2.14'

>     latestReleaseDate: 2023-08-29

Reviewer 2: ```suggestion
    latestReleaseDate: 2023-08-14
```

https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.2/index.html

-   releaseCycle: "4.1"
    releaseDate: 2020-07-21
    eol: 2022-10-31
    latest: '4.1.15'

>     latestReleaseDate: 2022-06-20

Reviewer 2: ```suggestion
    latestReleaseDate: 2022-06-09
```

https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.1/index.html

-   releaseCycle: "4.0"
    releaseDate: 2019-06-26
    eol: 2021-06-30
    latest: '4.0.14'

>     latestReleaseDate: 2021-06-22

Reviewer 2: ```suggestion
    latestReleaseDate: 2021-06-17
```

https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/4.0/index.html

-   releaseCycle: "3.2"
    releaseDate: 2018-06-25
    eol: 2020-09-30
    latest: '3.2.15'

>     latestReleaseDate: 2020-08-19

Reviewer 2: ```suggestion
    latestReleaseDate: 2020-08-11
```
https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/3.2/index.html

-   releaseCycle: "3.1"
    releaseDate: 2017-06-27
    eol: 2019-09-30
    latest: '3.1.11'

>     latestReleaseDate: 2019-03-01

Reviewer 2: ```suggestion
    latestReleaseDate: 2019-06-30 # https://web.archive.org/web/20190407030446/https://www.suse.com/lifecycle/
```

-   releaseCycle: "3.0"
    releaseDate: 2016-04-05
    eol: 2018-10-31
    latest: '3.0.12'

>     latestReleaseDate: 2018-06-01

Reviewer 2: ```suggestion
    latestReleaseDate: 2018-05-18
```

https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/3.0/index.html

---
title: SUSE Manager
category: server-app
tags: suse
iconSlug: suse
permalink: /suse-manager
versionCommand: zypper info SUSE-Manager-Server-release
releasePolicyLink: https://www.suse.com/lifecycle
changelogTemplate: https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/__RELEASE_CYCLE__/
releaseColumn: true
releaseDateColumn: true
eolColumn: General Support

> # Release dates are documented in each release notes in the "Version revision history" paragraph.

Reviewer 2: ```suggestion
# Release dates are documented in each release notes in the "Version revision history" paragraph.
# EOL dates can be found on https://www.suse.com/lifecycle.
```

---
title: SUSE Manager
category: server-app
tags: suse
iconSlug: suse
permalink: /suse-manager
versionCommand: zypper info SUSE-Manager-Server-release
releasePolicyLink: https://www.suse.com/lifecycle
changelogTemplate: https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/__RELEASE_CYCLE__/
releaseColumn: true
releaseDateColumn: true
eolColumn: General Support

# Release dates are documented in each release notes in the "Version revision history" paragraph.
# EOL dates can be found on https://www.suse.com/lifecycle.
releases:
-   releaseCycle: "5.0"
    releaseDate: 2024-07-12
    eol: 2026-06-30
    latest: '5.0.0'

>     latestReleaseDate: 2024-07-12

Reviewer 2: ```suggestion
-   releaseCycle: "5.0"
    releaseDate: 2024-07-16
    eol: 2026-06-30
    latest: '5.0.0'
    latestReleaseDate: 2024-07-16
```

According to https://www.suse.com/releasenotes/x86_64/SUSE-MANAGER/5.0/index.html#_version_revision_history

-   releaseCycle: "4.3"
    releaseDate: 2022-06-20
    eol: 2025-06-30
    latest: '4.3.13'
    latestReleaseDate: 2024-06-01

-   releaseCycle: "4.2"
    releaseDate: 2021-06-21
    eol: 2023-10-31
    latest: '4.2.14'
    latestReleaseDate: 2023-08-29

-   releaseCycle: "4.1"
    releaseDate: 2020-07-21
    eol: 2022-10-31
    latest: '4.1.15'
    latestReleaseDate: 2022-06-20

-   releaseCycle: "4.0"
    releaseDate: 2019-06-26
    eol: 2021-06-30
    latest: '4.0.14'
    latestReleaseDate: 2021-06-22

-   releaseCycle: "3.2"
    releaseDate: 2018-06-25
    eol: 2020-09-30
    latest: '3.2.15'
    latestReleaseDate: 2020-08-19

-   releaseCycle: "3.1"
    releaseDate: 2017-06-27

>     eol: 2019-09-30

Reviewer 2: ```suggestion
    eol: 2019-06-30
```

According to https://www.suse.com/lifecycle#suse-manager-for-retail-31 (not the same product, but plausible given the latest release date).

    latest: '3.1.11'
    latestReleaseDate: 2019-03-01

-   releaseCycle: "3.0"
    releaseDate: 2016-04-05

>     eol: 2018-10-31

Reviewer 2: ```suggestion
    eol: 2018-10-31 # https://web.archive.org/web/20190808082538/https://www.suse.com/lifecycle/
```

More in line with other releases, wdyt ?

Author: If we wan't to keep historic EOL dates it should be `2018-10-31` in accordance with https://web.archive.org/web/20190808082538/https://www.suse.com/lifecycle/.

Reviewer 2: Thanks, I did not thought to look to old versions of the page. Will add the link in the comments for future maintainers.

    latest: '3.0.12'
    latestReleaseDate: 2018-06-01

-   releaseCycle: "2.1"
    releaseDate: 2014-06-02

>     eol: 2017-04-30

Reviewer 2: ```suggestion
    eol: 2017-01-31 # https://web.archive.org/web/20170318160650/https://www.suse.com/lifecycle/
```

Author: Good point! It's long gone but I found an archived version here: https://web.archive.org/web/20170318160650/https://www.suse.com/lifecycle/

According this EOL was `2017-01-31`. Do we want to keep those historic information? 🙂

Reviewer 2: Yes, now that this is confirmed let's keep it.

    latest: '2.1.19'
    latestReleaseDate: 2016-10-31

-   releaseCycle: "1.7"
    releaseDate: 2012-08-09

>     eol: 2015-06-30

Reviewer 2: ```suggestion
    eol: 2015-06-30 # https://web.archive.org/web/20170318160650/https://www.suse.com/lifecycle/
```

Author: I doubled checked with an other version of the page: https://web.archive.org/web/20190808082538/https://www.suse.com/lifecycle/

So EOL should be `2015-06-30`, if we wan't to keep those historic information.
